data/avro-schema/src/main/resources/avro/net/corda/data/membership/command/registration/mgm/ProcessMemberVerificationResponse.avsc

```diff
@@ -4,5 +4,20 @@
   "namespace": "net.corda.data.membership.command.registration.mgm",
   "doc": "Command issued when a member responds to the member verification request send to the member by the MGM during registration.",
   "fields": [
+    {
```

Comment:
Same here as for `data/avro-schema/src/main/resources/avro/net/corda/data/membership/command/registration/mgm/VerifyMember.avsc`. This information could just be part of the registration state passed along with events.

Reply:
done